apps/console/src/app/pages/organizations/onboarding/OnboardingPage.tsx

```diff
@@ -39,7 +40,7 @@ export const OnboardingPage = () => {
     useUpdateCurrentUserMutation();
   const { mutateAsync: createProject, isLoading: isProjectCreationLoading } =
     useCreateProjectMutation();
-
+  usePageTitle(`Onboarding ${organization?.name || ""}`);
```

Comment:
```suggestion
  usePageTitle("Onboarding");
```
I think this is enough.